examples/04_glm_first_level/plot_adhd_dmn.py

```diff
@@ -56,6 +59,22 @@
 )
 seed_time_series = seed_masker.fit_transform(adhd_dataset.func[0])
 frametimes = np.linspace(0, (n_scans - 1) * t_r, n_scans)
+
+#########################################################################
+# Plot the time course of the seed region.
+import matplotlib.pyplot as plt
+
+fig = plt.figure(figsize=(15, 5))
```

Comment:
Can you make the figure ---much--- smaller ? For the moment, the labels, title and legend are dwarfed by the figsize.